src/core/components/physics_component.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+#include "src/core/common/bounding_box.h"
+
+struct PhysicsComponent {
+    BoundingBox bounding_box;

```

Comment:
Please, replace the BoundingBox with Dimension. Position should be passed to TransformationComponent -> see how the rendering system works. Then, if u want,  you can still get the physical boundingBox of entity by getting position from TransformationComponent and dimensions from PhysicsComponent

Reply:
Ok

Reply:
Ok